### ISSUE TITLE
ModuleNotFoundError: No module named 'tqdm.contrib.concurrent'

### ISSUE THREAD

**Issue by Y4rd13** [EricFillion/happy-transformer#244]
```python
!pip install happytransformer -q

from happytransformer import HappyWordPrediction, ARGS_WP_TRAIN
from happytransformer import HappyTextClassification, TCTrainArgs


---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-2-a3b0f90ccd90> in <module>()
----> 1 from happytransformer import HappyWordPrediction, ARGS_WP_TRAIN
      2 from happytransformer import HappyTextClassification, TCTrainArgs

17 frames
/usr/local/lib/python3.7/dist-packages/datasets/utils/file_utils.py in <module>()
     25 import posixpath
     26 import requests
---> 27 from tqdm.contrib.concurrent import thread_map
     28 
     29 from .. import __version__, config, utils

ModuleNotFoundError: No module named 'tqdm.contrib.concurrent'
```

**Comment by EricFillion** [comment]
Thanks for pointing this out! This was addressed in #243 and is now published under version 2.2.5

**Comment by Y4rd13** [comment]
> Thanks for pointing this out! This was addressed in #243 and is now published under version 2.2.5
No idea why colab was taking the 2.2.4 version, but now is working with 2.2.5
